### PR TITLE
listBlobEntriesAsync pattern

### PR DESCRIPTION
**Pull request by dehann**

### CODE REVIEW

**Review by Affie — APPROVED** [review]

**Review by Affie — APPROVED** [review]
Just fix the test. But looks on

File: src/navability/services/blobentry.py [hunk 1]
```diff
@@ -118,6 +120,12 @@ def listBlobEntries(
         entry['label'] for entry in entries
     ]
 
+async def listBlobEntries(
```
Comment (Affie) [inline src/navability/services/blobentry.py:123]: O, the async should not be ther

Comment (Affie) [inline src/navability/services/blobentry.py:123]:
```suggestion
def listBlobEntries(
```

Comment (dehann) [reply src/navability/services/blobentry.py]: oops right thanks

**Pull request merged**